libs/blocks/quiz-entry/quiz-entry.js

```diff
@@ -13,7 +14,7 @@ const App = ({
   debug = false,
 }) => {
   const [dataLoaded, setDataLoaded] = useState(false);
-  const [quizState, setQuizState] = useState({ userFlow: [], userSelection: [] });
+  const [quizState, setQuizState] = useState({ userFlow: [], userSelection: [], isML: false });
```

Comment:
What is the purpose of isML here? Can this be inferred from userSelection by the presence of ficodes, rather than standard card responses intead? It looks like you're carrying this all the way over to handleResultFlow in quiz/utils to determine if the analytics should include the "interest". 

Or better, can we just add this isML bit to userSelection, so we're adding less code elsewhere? you could drop all of the added ML stuff by just adding it to userSelection instead if mlInputUsed is true. userSelection is already passed to handleResultFlow so it'd just be a matter of intercepting it there. 

userSelection is made up of two objects, selectedQuestion, and selectedCards. Adding an isML object to userSelection should make it easier to get the information to handleResultFlow, without the need to include it separately in quizState.